wpilibc/src/main/native/cpp/DriverStation.cpp

```diff
@@ -28,6 +32,42 @@ struct MatchInfoData {
   int replayNumber = 0;
   DriverStation::MatchType matchType = DriverStation::MatchType::kNone;
 };
+
+class MatchDataSender {
+ public:
+  std::shared_ptr<nt::NetworkTable> table;
+  nt::NetworkTableEntry metaDataKey;
```

Comment:
Why are these called `*Key`? They're entries. We don't do Hungarian notation either, so the trailing "Key" can just be dropped from all of these (+ their Java equivalents)